instrumentation/log4j/log4j-2-testing/src/main/java/io/opentelemetry/instrumentation/log4j/v2_13_2/ListAppender.java

```diff
@@ -44,7 +47,12 @@ public void clearEvents() {
 
   @Override
   public void append(LogEvent logEvent) {
-    events.add(logEvent);
+    // Event object may be reused by the framework so copy the data we need.
```

Comment:
Never knew log4j has such an optimization, too bad most spring users never get to see it :D